### ISSUE TITLE
Fine-Tuning on VGG Diverges

### ISSUE THREAD

**Issue by RijulGupta-DM** [ai-forever/ghost#61]
I've been attempting to fine-tune this model using the instructions provided, but the generated images seem to diverge after just a few thousand steps. See the attached image for generated samples
![image](https://user-images.githubusercontent.com/104281028/211169524-ff95e11d-0717-424a-b167-0b24057615e1.png)

I've experimented with reducing/turning off all losses one-by-one. For example

Default - Diverges
weight_adv = 1
weight_attr = 10
weight_id = 20
weight_rec = 10
weight_eyes = 0

No Attr - Diverges
weight_adv = 1
weight_attr = 0
weight_id = 20
weight_rec = 10
weight_eyes = 0

No ID - Diverges
weight_adv = 1
weight_attr = 10
weight_id = 0
weight_rec = 10
weight_eyes = 0

No Reconstruction - Diverges
weight_adv = 1
weight_attr = 10
weight_id = 20
weight_rec = 0
weight_eyes = 0

The only run that produced decent results was when Adversarial loss was completely turned off. But turning off adversarial loss has it's own problems (see attached image)
weight_adv = 1
weight_attr = 10
weight_id = 20
weight_rec = 10
weight_eyes = 0

![image](https://user-images.githubusercontent.com/104281028/211169563-27735297-9b91-4c07-887c-f1612eb4254e.png)

I've experimented with setting discr_force as True, this did not help produce good results.

I've experimented with reducing LR to 4e-5 (10% of default) and adding LR schedulers. This helped somewhat, but the model would still diverge after 10K-20K steps. 

Was curious if you had any other suggestions - any help would be greatly appreciated!

Great work on this paper - very eager to see if we can fine-tuning to work so I can play around with some additional loss functions. I can keep sharing results here if that's helpful!

**Comment by hichemfelouat** [comment]
Hi RijulGupta-DM 
does the code work on Anaconda?

**Comment by RijulGupta-DM** [comment]
@hichemfelouat ,

The code is set up using Anaconda. Everything is set up properly and it trains without crashing. The generated images just don't look good.

![image](https://user-images.githubusercontent.com/104281028/211241167-25403d1e-0738-40f7-a7ac-0c864d19d387.png)

**Comment by hichemfelouat** [comment]
@RijulGupta-DM 
Please could you export your anaconda environment for this project and then send it to me as a ymal file?

**Comment by fswienty** [comment]
@RijulGupta-DM Since you've marked this as completed, could you elaborate on how you solved your issues? I'm running into similar problems, any help would be much appreciated.
